test/buildenv.sh

```diff
@@ -12,7 +12,7 @@ ARM_EMU_BIN="${ARM_EMU_BIN:-}"
 BINFMT_MISC="${BINFMT_MISC:-/proc/sys/fs/binfmt_misc/}"
 
 FILESYSTEMS="ext4 f2fs overlay squashfs tmpfs"
-COMMANDS="apk blkid fsck.ext4 fsck.f2fs losetup mkfs.ext4 mkfs.f2fs mksquashfs sed sfdisk shuf xz"
+COMMANDS="apk blkid dpkg fsck.ext4 fsck.f2fs gettext losetup mkfs.ext4 mkfs.f2fs mksquashfs sed sfdisk shuf tar xz"
```

Comment:
Don't test for the gettext command, test for the 'envsubst' command, as that is the command that you need.

Also, no test was added for the missing features of tar that dpkg 'depends' on. Tar will always be available as it's part of busybox.

Reply:
Busybox tar won't work for dpkg, so the dpkg package incorrect doesn't depend on gnu-tar, but needs it.